libs/threadmanager/CMakeLists.txt

```diff
@@ -45,3 +45,9 @@ add_hpx_module(threadmanager
     hpx_topology
   CMAKE_SUBDIRS examples tests
 )
+
+if (TARGET hpx::apex)
+  target_link_libraries(hpx_threadmanager PRIVATE hpx::apex)
+  get_target_property(_apex_inc_dir hpx::apex INTERFACE_INCLUDE_DIRECTORIES)
+  target_include_directories(hpx_compute INTERFACE "${_apex_inc_dir}")
```

Comment:
Why not just add `hpx::apex` as a dependency to `hpx_compute` as well?

Reply:
`hpx_compute` already depends on APEX. I'll add it to `hpx_compute_cuda` though.